apps/mobile/src/components/Community/CommunityHeader.tsx

```diff
@@ -34,7 +34,8 @@ export function CommunityHeader({ communityRef }: Props) {
     bottomSheetRef.current?.present();
   }, [hasCommunityDescription]);
 
-  const formattedDescription = community.description?.replace(/\n/g, ' ');
+  const cleanedSetences = community.description?.trim().replace(/\s+/g, ' ');
+  const formattedDescription = cleanedSetences?.split(/[.!?]\s+/).join(' ');
```

Comment:
nit:
1. `cleanedSentences*`
2. for code that looks like dark magic, we typically like to add an explanatory comment. could we add one above L37 explaining what the purpose of this is?